test/runtests.jl

```diff
@@ -7,44 +7,5 @@ include(joinpath(pathof(AbstractAlgebra), "..", "..", "test", "Rings-conformance
 # Used in both Rings/slpolys-test.jl and StraightLinePrograms/runtests.jl
 const SLP = Oscar.StraightLinePrograms
 include("printing.jl")
-
-include("PolyhedralGeometry/runtests.jl")
-include("Combinatorics/runtests.jl")
-
-include("GAP/runtests.jl")
-include("Groups/runtests.jl")
-
 include("Rings/runtests.jl")
```

Comment:
Please try this:
```suggestion
include("Rings/runtests.jl")
Singular.libSingular.set_option("OPT_REDSB", false);
```
`integral_basis` seems to change some singular flags, and some other function in the removed testsets seems to reset this later one.

Also, it might be better to compare the ideals in the tropical test instead of the generators if that is not too expensive.

Edit: the suggested change is just meant for testing and not as a fix.

Reply:
Thanks for investigating this. With the next singular, we hopefully won't need these kinds of hacks.